src/WebJobs.Extensions.Http/Shim/HttpRequestMessageFeature.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#if NET6_0_OR_GREATER
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Http.Shim
+{
+    public class HttpRequestMessageFeature : IHttpRequestMessageFeature
```

Comment:
this code is copied here from the `Microsoft.AspNetCore.Mvc.WebApiCompatShim` package: 
ref: https://github.com/aspnet/Mvc/blob/b18526cdc8d8c868e8d7f9c6c8484064215e1002/src/Microsoft.AspNetCore.Mvc.WebApiCompatShim/HttpRequestMessage/HttpRequestMessageFeature.cs

in case somebody somehow manages to call this code path